cmd/kubeadm/app/cmd/reset.go

```diff
@@ -139,6 +140,10 @@ func (r *Reset) Run(out io.Writer, client clientset.Interface) error {
 	etcdDataDir, err := getEtcdDataDir(etcdManifestPath, client)
 	if err == nil {
 		dirsToClean = append(dirsToClean, etcdDataDir)
+		if err := removeEtcdMember(client); err != nil {
+			klog.Errorf("[reset] failed to remove etcd member from the cluster: %+v", err)
+			return err
```

Comment:
Since `reset` is a best effort and we cannot assume anything about the current state of the system I'd say we don't want to `return` here. Log the error, but let's continue with the `reset` execution, so we can clean as much as possible.

Reply:
+1

Reply:
Thanks for you response. There's a question that if remove etcd member error, maybe make etcd cluster not health, what to do in this case?

Reply:
IMO you should expect that it can happen if you are removing a control-plane

Reply:
If it happens, kubeadm reset workflow continue or aborted? Do we have a method deal with it?

Reply:
Should continue, because we want to clean as much as possible.

Reply:
Should we add some tips in log that tell user how to fix it if remove etcd member error?

Reply:
it would be nice, if it's a simple message.

Reply:
I update the code, and remove return error if something wrong.

Reply:
Thanks! Can you remove the wip and squash your commits?

Reply:
I want to add some testing. When testing done, i squash commits and remove the wip, is it ok?

Reply:
Yes, Go ahead!